src/site/_includes/components/CodePattern.js

```diff
@@ -16,18 +15,18 @@ module.exports = (patternId, height = 300) => {
         Prism.languages[type],
         type,
       );
-      return html`
+      return `
         <web-tab title="${asset.type}" data-label="${asset.type}">
           <pre><code class="language-${asset.type}">${content}</code></pre>
         </web-tab>
       `;
     })
     .join('\n');
 
-  return html`<div class="code-pattern flow flow-space-400">
+  return `<div class="code-pattern flow flow-space-400">
     <div class="code-pattern__content" style="height: ${height}px;">
       <div class="code-pattern__demo">
```

Comment:
These elements need to stack vertically on mobile.

Reply:
I'd rather do it in a separate Pr, as this one is about suits&sets pages rather than CodePattern element.

Reply:
Here's an issue to track: https://github.com/GoogleChrome/web.dev/issues/5979